easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'Bundle'
```

Comment:
@verdurin You should use `PythonBundle` here (tests trip over that now)

Reply:
Done.